spec/features/admin/index_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe "Admin Dashboard Index Page" do
+  before(:each) do
+
+  end
+
+  describe "Admin Dashboard Display" do
+    
+    # User Story 19
+    it "displays a header indicating that it is the admin dashboard" do
+      visit admin_path
```

Comment:
excellent use of the admin path. thank you for communicating this to us. This should be the standard we use for the project!